### DISCUSSION TITLE
Saving the Tabs in Sequence

### DISCUSSION THREAD

**Discussion by touwys** [gildas-lormeau/SingleFile#1596] [Ideas]
I surmise that this idea has long been considered, and then it was probably ditched for dearth of speed — or some such other practical reason. That said, however, could such a feature be provided as an option for those who would willingly sacrifice speed for practicality? Enabling this option will allow the saving of the open tabs in the sequence they are regularly sorted — using for e.g. — add-ons such as the one below for Firefox.

![image](https://github.com/user-attachments/assets/b71a6f9b-49c0-415c-89e1-55691c6f8fd0)
 
Thank you.

**Comment by gildas-lormeau** [comment]
Sorry for the very late reply. You should be able to keep the tab order if you set the hidden option `maxParallelWorkers` to `1`, see https://github.com/gildas-lormeau/SingleFile/wiki/Hidden-options.

**Comment by touwys** [comment]
Thank you, no problem. I have now altered `maxParallelWorkers` from 8 (default) to 1. Testing.
